rush-plugins/rush-audit-cache-plugin/tsconfig.json

```diff
@@ -35,7 +35,7 @@
       "node"
     ] /* Specify type package names to be included without being referenced in a source file. */,
     // "allowUmdGlobalAccess": true,                     /* Allow accessing UMD globals from modules. */
-    // "resolveJsonModule": true,                        /* Enable importing .json files */
+    "resolveJsonModule": true,                        /* Enable importing .json files */
```

Comment:
Why you need turn this on?

Reply:
In the previous implementation, a json file was defined separately, which is not needed now